opendbc/safety/tests/libsafety/safety_helpers.h

```diff
@@ -12,7 +12,16 @@ bool safety_config_valid() {
     const RxCheck addr = current_safety_config.rx_checks[i];
     bool valid = addr.status.msg_seen && !addr.status.lagging && addr.status.valid_checksum && (addr.status.wrong_counters < MAX_WRONG_COUNTERS) && addr.status.valid_quality_flag;
     if (!valid) {
-      // printf("i %d seen %d lagging %d valid checksum %d wrong counters %d valid quality flag %d\n", i, addr.status.msg_seen, addr.status.lagging, addr.status.valid_checksum, addr.status.wrong_counters, addr.status.valid_quality_flag);
+      printf("i %d seen %d lagging %d valid checksum %d wrong counters %d valid quality flag %d\n",
+      i, addr.status.msg_seen, addr.status.lagging, addr.status.valid_checksum, addr.status.wrong_counters, addr.status.valid_quality_flag);
+      for (int j = 0; j < MAX_ADDR_CHECK_MSGS; j++) {
+        const CanMsgCheck msg = addr.msg[j];
+        if (msg.addr == 0) {
+          break;
+        }
+        printf("i %d j %d bus %d addr 0x%x len %d\n",
+        i, j, msg.bus, msg.addr, msg.len);
+      }
```

Comment:
if you split this out and put it behind an environment variable (like `LOGPRINT=debug`), we can merge this